osu.Game/Screens/Menu/SongTicker.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Game.Graphics.Sprites;
+using osuTK;
+using osu.Game.Graphics;
+using osu.Framework.Bindables;
+using osu.Game.Beatmaps;
+using osu.Framework.Localisation;
+
+namespace osu.Game.Screens.Menu
+{
+    public class SongTicker : Container
+    {
+        private const int fade_duration = 800;
+
+        [Resolved]
+        private Bindable<WorkingBeatmap> beatmap { get; set; }
+
+        private readonly Bindable<WorkingBeatmap> workingBeatmap = new Bindable<WorkingBeatmap>();
```

Comment:
this is not how bindables work. you do not need this field.

Reply:
is this due to `SongTicker` never getting removed?

Reply:
In one pr someone told me to use local bindable rather than using `BindValueChanged` for resolved one.

Reply:
https://github.com/ppy/osu-framework/blob/master/osu.Framework/Allocation/ResolvedAttribute.cs#L115